src/Connection.ts

```diff
@@ -1255,14 +1255,16 @@ export class Connection<
 
 	/**
 	 * Query a predefined object view.
+	 * @param design The namespace of the object view, as defined in io-package.json. Usually the adapter name, e.g. "hm-rpc"
+	 * @param type The type of object.
 	 * @param start The start ID.
 	 * @param end The end ID.
-	 * @param type The type of object.
 	 */
 	getObjectView<T extends ioBroker.ObjectType>(
-		start: string,
-		end: string,
-		type: T,
+		design: 'system' | 'chart' | string,
```

Comment:
The type `'system' | 'chart' | string` is just `string`.
![grafik](https://user-images.githubusercontent.com/17641229/148061690-e21c744f-932c-4aa3-bd1c-4880614cde29.png)

Reply:
(which isn't very helpful and doesn't do what you intended)

Reply:
Yeah, but I don't know wich names here are usefull... what is mostly used

Reply:
I'd leave the current method as is and add another one where `design` and `view` is just typed as `string`.